latest/schemas/strict_image.schema

```diff
@@ -0,0 +1,19 @@
+{
+  "$id": "https://ngff.openmicroscopy.org/0.4/schemas/strict_image.schema",
+  "allOf": [
+    {
+      "$ref": "https://ngff.openmicroscopy.org/0.4/schemas/image.schema"
```

Comment:
Do these IDs get updated directly to `/0.5/` at this point? (don't need to use `/latest/`, right?)
NB: I notice that https://ngff.openmicroscopy.org/0.4/schemas/image.schema is 404, so this schemas won't work without the custom resolver used in the tests.

Reply:
No you are correct,  I didn't do the update work as I was waiting to get a consensus about the value of adding these scneas.

A few comments:
- definitely need to replace `0.4`. I would not use `0.5`  as that would communicate a released schema and that has the potential of creating confusion. `latest` would be an option or maybe something that explicitly says this is a development schema e.g. `0.5dev`?
- the individual `version` elements in the JSON snippets/examples will likely need to be updated to follow the same convention
- [the release process](https://github.com/ome/ngff/blob/main/README.md#release-process) will need to be amended to include these steps